TelenorNBIoT.cpp

```diff
@@ -425,21 +435,43 @@ bool TelenorNBIoT::isError(const char *line)
 
 int TelenorNBIoT::splitFields(char *line, char **fields)
 {
-    int found = 0;
+    int found = 1;
     char *p = line;
     fields[0] = line;
-    while (*p++)
+    while (*++p)
```

Comment:
You were right to think we should avoid this style of incrementing, even though it is idiomatic.  It is clearer to do the increment separate from the check.